wrappers/here.rb

```diff
@@ -457,10 +454,13 @@ def here_strict_restriction(param_value)
     end
 
     ##
-    # dist: value in meters
-    # 100km: 7, 1200km: 2, 1400km: 1
-    def coef_distance(dist)
-      7 - [(dist.ceil / 1000) / 201, 6.0].min
+    # distance > 2000km : 5
+    def max_row(dist_km)
+      if dist_km <= 1_000
+        15
+      else # 1500km: 10, 2000km : 5
+        [-0.01 * dist_km + 25, 1].max.floor
```

Comment:
```
if dist_km <= 1_000
  15
elsif dist_km >= 2_000
  5
else # 1000km: 15, 1500km: 10, 2000km : 5
  25 - dist_km / 100
end
```
is more readable

Reply:
The code you propose will return a minimum of 5, on huge diagonals like 3000km and more, I thought we wanted to do matrices of 100x1 like the test does. On the diagonal of 2000km a 5x was more interesting than 5 times a 1x or 2x.